### PR TITLE
Add aws-account-operator service

### PR DESCRIPTION
**Pull request by jharrington22**
This PR adds the deploy configuration for the aws-account-operator

### CODE REVIEW

File: aws-account-operator-services/aws-account-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 13c36e34cec72a9e416b7a48f75cc7586f23c897
+  name: aws-account-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```
Comment (jmelis) [inline aws-account-operator-services/aws-account-operator.yaml:4]: this path doesn't seem to exist

**Review by jmelis — CHANGES_REQUESTED** [review]

File: aws-account-operator-services/aws-account-operator.yaml (hunk 1 above)
Comment (jharrington22) [reply aws-account-operator-services/aws-account-operator.yaml:4]: I have a branch in the aws-account-operator repo which contains the scripts just cleaning them up.

Comment (jharrington22) [reply aws-account-operator-services/aws-account-operator.yaml:4]: That PR is ready for review/merge https://github.com/openshift/aws-account-operator/pull/38

**Comment by jmelis** [comment]
LGTM, but would like to wait until this is merged: https://github.com/openshift/aws-account-operator/pull/38#pullrequestreview-225483256

**Review by jmelis — APPROVED** [review]

**Pull request merged**